README.md

```diff
@@ -33,6 +33,14 @@ See examples below:
 {{/let}}
 ```
 
+**Inline use**
+Note: requires Ember 2.0+ (ie. does not support 1.13)
+
+```hbs
+{{let greeting (concat "hello " to)}}
+{{greeting}} - <button {{action (action (mut to) "world")}}>Greet the world!</button>  
+```
```

Comment:
If there's any pushback against inline use, I'd see it coming in the form of "hey, isn't assignment a side-effect?" I think we can head this off at the pass by explicitly pointing out that no, in fact, this is just short hand for the block form.

That does make me think though, is there some way to mark the stream and then issue a warning if you're assigning to the same path twice?